src/docs/events.yaml

```diff
@@ -0,0 +1,204 @@
+paths:
+  /events/{id}:
+    get:
+      tags: 
+      - "Events"
+      summary: "Get event by id"
+      description: "Returns a single event"
+      operationId: "getEventById"
+      produces:
+      - "application/json"
+      parameters:
+        - name: "id"
+          in: "path"
+          description: "ID of the event to return"
+          required: true
+          type: "integer"
+          format: "int64"
+      responses:
+        "200":
+          description: "succesful operation"
+          schema:
+            $ref: "#/components/schemas/Event"
+        "404":
+          description: Invalid event ID supplied, Event not found
+        "500": 
+          description: Internal server error
+components:
+  schemas:
+    Event:
+     type: "object"
+     required:
+       - "title"
+       - "content"
+       - "coverImage"
+       - "date"
+       - "category"
+       - "address"
+       - "location"
+     properties:
+       publisherId:
+        type: "integer"
+        format: "objectId"
+       title:
+        type: "string"
+        description: "Event Title"
+       content:
+        type: "string"
+       coverImage:
+        type: "string"
+       date:
+        type: "date"
+       category:
+        type: "string"
+        enum:
+         - "No Poverty"
+         - "Zero Hunger"
+         - "Good Health And Well-Being"
+         - "Quality Education"
+         - "Gender Equality"
+         - "Clean Water And Sanitation"
+         - "Affordable And Clean Energy"
+         - "Animals"
+         - "Oceans"
+         - "Nature"
+         - "Reduced Inequalities"
+         - "Sustainable Cities And Communities"
+         - "Responsible Consumption And Production"
+         - "Climate Action"
+         - "Life Below Water"
+         - "Life On Land"
+         - "Peace"
+         - "Youth"
+         - "Justice"
+       confirmedVolunteers:
+        type: "array"
+        items:
+         type: "object"
+         $ref: "#components/schemas/user"
```

Comment:
1. **#/components/schemas/user**
2. After you populate the users as I suggested, you have to change this to a ShortUser schema: 
```
ShortUser:
  properties:
    _id:
      type: string
      example: 62e980586aca79a3936917e6
    firstName:
      type: string
      example: John
    lastName:
      type: string
      example: Doe
    email:
      type: string
      example: mail@example.com
    __t:
      type: string
      example: User
    profileImage:
      type: string
      example: https://www.example.com/image.jpg
    fullName:
      type: string
      example: John Doe
    id:
      type: string
      example: 62e980586aca79a3936917e6

```